src/Elasticsearch.Net/Api/RequestParameters/RequestParameters.NoNamespace.cs

```diff
@@ -1798,13 +1784,6 @@ public class SearchTemplateRequestParameters : RequestParameters<SearchTemplateR
 			set => Q("allow_no_indices", value);
 		}
 
-		///<summary>Indicates whether network round-trips should be minimized as part of cross-cluster search requests execution</summary>

```

Comment:
breaking change

Reply:
yes, looks to have been removed from the spec:

- 7.3: https://github.com/elastic/elasticsearch/blob/7.3/rest-api-spec/src/main/resources/rest-api-spec/api/search_template.json#L78-L83
- 7.4: https://github.com/elastic/elasticsearch/blob/7.4/rest-api-spec/src/main/resources/rest-api-spec/api/search_template.json